list

```diff
@@ -177,18 +179,22 @@ bigcc.cc
 bigfi.sh
 biggo.tw
 biibly.com
+bin.wf
 binged.it
 bit.ly
 bitly.com
 bitly.is
 bitly.lc
+bitly.pw
 bitly.ws
+bitshortly.com
 bl.ink
 bl1.ir
 blap.net
 blbrd.cm
 blck.by
 blizz.ly
+blnk.in
```

Comment:
This one doesn't look like working?

Reply:
It's interesting that they doesn't seem to handle the protocol properly...

Reply:
Old link expired/not found.

https://blnk.in/s1jng
Archive: https://archive.is/fKqlc
Target: `https://github.com/PeterDaveHello/url-shorteners`

Reply:
hmmm... I doubt the links on this platform expire very soon, then it's expired again :rofl: